### ISSUE TITLE
oas3-valid-content-schema-example seems to misbehave with arrays

### ISSUE THREAD

**Issue by bkabrda** [stoplightio/spectral#927]
**Describe the bug**

Given following schema:

```yaml
components:
  schemas:
    X:
      type: array
      items:
        type: string
      example:
        value:
          - "foo"
```

Validating yields error `error  oas3-valid-content-schema-example  `example` property type should be array`. When I transform it to

```yaml
components:
  schemas:
    X:
      type: array
      items:
        type: string
      example:
        - "foo"
```

I get the following warning: `warning  example-value-or-externalValue  Example should have either a `value` or `externalValue` field.`

**Expected behavior**

I think at least the first example is correct according to the openapi spec (so shouldn't yield an error), not sure about the second one.

**Environment (remove any that are not applicable):**
Using Docker image stoplight/spectral:5 (image id 38270ed7d5da).

**Comment by m-mohr** [comment]
The first example is invalid, the second is valid according to the OAS3 spec.
Still, spectral had a bug, which has been fixed just recently, see issue #883 .
Please note that there's a difference between "example" and "examples" in OAS.

**Comment by P0lip** [comment]
Hello!
I believe this issue has been already fixed.
As, Matthias mentioned, the first example is invalid, yet the second is in fact perfectly correct and Spectral won't report any errors for such usage anymore.

If you still face the issue, please do not hesitate to re-open the ticket.
